provisioning/containers/pycharm/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:20.04
+
+ARG PYCHARM_VERSION="2020.2.3"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install wget and other required libraries for pycharm to be rendered
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends libxrender1 libxtst6 libxi6 libfontconfig1 \
+    && apt-get install -y wget python3 python3-venv \
+    && apt-get autoclean -y \
+    && apt-get clean -y
+
+# Download and extract Pycharm
+RUN mkdir -p /opt/pycharm && \
+    wget -qO- https://download.jetbrains.com/python/pycharm-community-${PYCHARM_VERSION}.tar.gz | tar xz --strip 1 -C /opt/pycharm
+
+# Clean up
+RUN apt-get purge wget -y && apt-get --purge autoremove -y && apt-get autoclean -y

```

Comment:
Yes, it's actually like that, and specifically that layer introduces ~1 MB. However, the reason for this layer was to remove `wget` to prevent users from leveraging it and thus being able to download software. What do you suggest? To keep it or to remove it?

Reply:
Ok, I got you point and I agree that the difference is really insignificant.  Probably the second alternative is a bit more Docker style, but also the other is totally fine

Reply:
Thank you for looking at it. I will follow your suggestion to remove the clean up RUN layer and I will integrate it in the same RUN layer where `wget` is installed and used to download the archive, as per the suggestion above.